Gemfile

```diff
@@ -72,15 +73,15 @@
   end
 
   gem 'avalon-about', git: "https://github.com/avalonmediasystem/avalon-about.git", tag: 'avalon-r4'
-  gem 'about_page', git: "https://github.com/avalonmediasystem/about_page.git", tag: 'avalon-r4'
+  gem 'about_page', git: "https://github.com/avalonmediasystem/about_page.git"
```

Comment:
This eventually needs to have a tag because the latest about_page revs in master are for avalon-6 only.